csrankings.csv

```diff
@@ -827,9 +829,11 @@ Andrea Lockerd,Georgia Institute of Technology,http://www.cc.gatech.edu/~athomaz
 Andrea Lockerd Thomaz,Georgia Institute of Technology,http://www.cc.gatech.edu/~athomaz,jIs-Y2gAAAAJ
 Andrea Maggiolo Schettini,University of Pisa,http://www.di.unipi.it/~maggiolo,NOSCHOLARPAGE
 Andrea Marino,University of Pisa,http://pages.di.unipi.it/marino,FUKaLzkAAAAJ
+Andrea Pietracaprina,University of Padova,http://www.dei.unipd.it/~capri,X4HU5HIAAAAJ
 Andrea S. LaPaugh,Princeton University,https://www.cs.princeton.edu/~aslp,NOSCHOLARPAGE
 Andrea Tagliasacchi,University of Victoria,http://gfx.uvic.ca,1RmD-YsAAAAJ
 Andrea Thomaz,Georgia Institute of Technology,http://www.cc.gatech.edu/~athomaz,jIs-Y2gAAAAJ
+Andrea Vedaldi,University of Oxford,http://www.robots.ox.ac.uk/~vedaldi,bRT7t28AAAAJ

```

Comment:
Robotics is not CS. See above comment. This person and others below are not listed as faculty in CS (http://www.cs.ox.ac.uk/people/faculty.html).

Reply:
Added A. Vedaldi and A. Zisserman both from robotics institute at Oxford which has a huge intersection with CS (a similar example is the robotic dept. at CMU). They advise plenty of PhD students from CS and they are two of the most important people in computer vision and machine learning worldwide.

Reply:
Unless they are able to _solely_ advise students _in CS_, they do not meet the inclusion criteria. That is the case for members of the Robotics dept at CMU.